docker/docker-compose.yml

```diff
@@ -9,6 +9,9 @@ services:
   tanner_redis:
     build: ./redis
     container_name: tanner_redis
+    privileged: true
```

Comment:
is this required?

Reply:
privileged: true does not seem to be required.

Reply:
then please remove 👍